packages/legend-application-studio/src/components/editor/side-bar/ProjectOverview.tsx

```diff
@@ -19,46 +19,80 @@ import { observer } from 'mobx-react-lite';
 import { LEGEND_STUDIO_TEST_ID } from '../../LegendStudioTestID.js';
 import {
   clsx,
+  AssistantIcon,
```

Comment:
Overall, this file should be moved out of `src/components/editor/side-bar` and maybe into `src/components/dashboard`, again, please review the purpose of this PR in https://github.com/finos/legend-studio/issues/1523